2_process/src/process_and_style.R

```diff
@@ -1,17 +1,17 @@
-process_data <- function(site_data_filename){
-  nwis_data<- read_csv(site_data_filename)
-  nwis_data_clean <- rename(nwis_data, water_temperature = X_00010_00000) %>% 
+process_data <- function(site_data, fileout){
+  nwis_data_clean_csv <- rename(site_data, water_temperature = X_00010_00000) %>% 
     select(-agency_cd, -X_00010_00000_cd, -tz_cd)
-  
-  return(nwis_data_clean)
+  write_csv(nwis_data_clean_csv, fileout)
+  return(fileout)
 }
 
-annotate_and_style_data <- function(site_data_clean, site_filename){
-  site_info <- read_csv(site_filename)
-  annotated_data <- left_join(site_data_clean, site_info, by = "site_no") %>% 
+annotate_and_style_data <- function(filein, site_info, fileout){
+  site_data_csv<- read_csv(filein)
+  annotated_data_csv <- left_join(site_data_csv, site_info, by = "site_no") %>% 
     select(station_name = station_nm, site_no, dateTime, water_temperature, latitude = dec_lat_va, longitude = dec_long_va)%>%
     mutate(station_name = as.factor(station_name))
+    write_csv(annotated_data_csv, fileout)
```

Comment:
I think maybe there is a bit of confusion on when to use `_csv` in the naming. This should be used in the `_targets.R` files when the target you are creating is a file and so the output is a string of the filepath (when you do `tar_load([target_name])` the resulting object in your environment is something like `"out/my_file.csv"`). Internally to the function, I think your `filein` and `fileout` conventions are great. 

In both of these lines where you have `site_data_csv` and `annotated_data_csv` as objects, those objects actually contain the data - both are data.frames. By naming them with `_csv` here, you are implying that they contain the filepath but they contain the data! So, in both of these cases, you should drop `_csv`.